x/gov/keeper/tally.go

```diff
@@ -118,6 +118,13 @@ func (keeper Keeper) Tally(ctx sdk.Context, proposal types.Proposal) (passes boo
 		return true, false, tallyResults
 	}
 
+	// if the proposal is expedited and less than expedited_quorum_threshold, the proposal fails
+	if proposal.IsExpedited {
```

Comment:
Also, can we have either a unit or a functional test validating these test cases:
1. proposal is expedited, proposal passes when quorum is greater than expedited
2. proposal is expedited, proposal fails when quorum is lower than expedited quorum but higher than regugular quorum
3. proposal is expeditd, proposals fails when quorum is lower than regular quorum
4. proposal is regular, proposal passes when quorum is greater than regular quorum but lower than expedited quorum

Reply:
added test
1. https://github.com/osmosis-labs/cosmos-sdk/pull/457/files#diff-ed4e03a1684493ef5faebd259fba6adbd95295a1e8e3cecd54c4f27e30de7da9R491
2. https://github.com/osmosis-labs/cosmos-sdk/pull/457/files#diff-ed4e03a1684493ef5faebd259fba6adbd95295a1e8e3cecd54c4f27e30de7da9R513
3. https://github.com/osmosis-labs/cosmos-sdk/pull/457/files#diff-ed4e03a1684493ef5faebd259fba6adbd95295a1e8e3cecd54c4f27e30de7da9R543
4. https://github.com/osmosis-labs/cosmos-sdk/pull/457/files#diff-ed4e03a1684493ef5faebd259fba6adbd95295a1e8e3cecd54c4f27e30de7da9R572